### PR TITLE
PLT-7360 Adding SSE to compliance

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: webapp/components/admin_console/storage_settings.jsx [hunk 1]
```diff
@@ -113,6 +114,29 @@ export default class StorageSettings extends AdminSettings {
...
                 />
             );
+
+            amazonSSEComp =
+                (
+                    <BooleanSetting
+                        id='amazonS3SSE'
+                        label={
+                            <FormattedMessage
+                                id='admin.image.AmazonSSETitle'
```
Comment (saturninoabril) [inline webapp/components/admin_console/storage_settings.jsx]: @coreyhulen might need to update id to `admin.image.amazonS3SSETitle`

File: webapp/components/admin_console/storage_settings.jsx [hunk 2]
```diff
@@ -113,6 +114,29 @@ export default class StorageSettings extends AdminSettings {
...
+                                defaultMessage='Enable Server-Side Encryption for Amazon S3:'
+                            />
+                        }
+                        placeholder={Utils.localizeMessage('admin.image.AmazonSSEExample', 'Ex "false"')}
+                        helpText={
+                            <FormattedHTMLMessage
+                                id='admin.image.AmazonSSEDescription'
```
Comment (saturninoabril) [inline webapp/components/admin_console/storage_settings.jsx]: same here `admin.image.amazonS3SSEDescription`

File: webapp/components/admin_console/storage_settings.jsx [hunk 3]
```diff
@@ -113,6 +114,29 @@ export default class StorageSettings extends AdminSettings {
...
+                                defaultMessage='When true, encrypt files in Amazon S3 using server-side encryption with Amazon S3-managed keys. See <a href="https://about.mattermost.com/default-server-side-encryption" target="_blank">documentation</a> to learn more.'
```
Comment (saturninoabril) [inline webapp/components/admin_console/storage_settings.jsx]: might need to be 
```
defaultMessage='When true, encrypt files in Amazon S3 using server-side encryption with Amazon S3-managed keys. See <a href=\"https://about.mattermost.com/default-server-side-encryption\">documentation</a> to learn more.'
```

File: webapp/components/admin_console/storage_settings.jsx (hunk 1 above)
Comment (coreyhulen) [inline webapp/components/admin_console/storage_settings.jsx]: I didn't change or alter those from the community PR, just the ID.

File: webapp/components/admin_console/storage_settings.jsx (hunk 2 above)
Comment (coreyhulen) [inline webapp/components/admin_console/storage_settings.jsx]: I didn't change or alter those from the community PR, just the ID.

File: webapp/components/admin_console/storage_settings.jsx (hunk 1 above)
Comment (coreyhulen) [inline webapp/components/admin_console/storage_settings.jsx]: ugg...wait nvm.  I just realized the original PR didn't have them correctly.  I'll submit a fix soon.

File: webapp/components/admin_console/storage_settings.jsx (hunk 2 above)
Comment (coreyhulen) [inline webapp/components/admin_console/storage_settings.jsx]: ugg...wait nvm.  I just realized the original PR didn't have them correctly.  I'll submit a fix soon.

**Comment by coreyhulen** [comment]
@saturninoabril loc ids are updated, feel free to merge with the next RC.

**Pull request merged**